src/comestible_inv.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+#ifndef comestible_inv_H
+#define comestible_inv_H
+#include "cursesdef.h"
+#include "comestible_inv_pane.h"
+
+#include <array>
+#include <string>
+#include <vector>
+
+class uilist;
+class vehicle;
+class item;
+
+
+void comestible_inv( int bio = -1 );
+// see item_factory.h
+class item_category;
+
+class comestible_inventory
+{
+    public:
+        comestible_inventory();
+        ~comestible_inventory();
+
+        virtual void display();
+    protected:
+        virtual input_context register_actions();
+        virtual std::string process_actions( input_context ctxt );
+
+        const int head_height;
+        const int min_w_height;
+        const int min_w_width;
+        const int max_w_width;
+
+        // minimap that displays things around character
+        catacurses::window minimap;
+        catacurses::window mm_border;
+        const int minimap_width = 3;
+        const int minimap_height = 3;
+        void redraw_minimap();
+        char minimap_get_sym() const;
+
+        int w_height;
+        int w_width;
+
+        int headstart;
+        int colstart;
+
+        bool recalc;
+        bool redraw;
+
+        comestible_inventory_pane *pane;
```

Comment:
Please don't use raw owning pointer. Make this a `std::unique_ptr` and use `make_unique` to initialize it and get rid of the `delete pane` statement.